spec/data/02_unimplemented_path.json

```diff
@@ -16,6 +16,7 @@
     "application/json"
   ],
   "paths": {
+    "x-something": "Vendor Specific Tag",
```

Comment:
this should probably be in the "happy case" well formed data file:  spec/data/sample2.0.json

The file name should probably start with `01_` to make it clearer. The numbered scenarios are specific badly or oddly formed swagger files to test apivores behaviour in negative scenarios.